src/OmniSharp.LanguageServerProtocol/Handlers/OmniSharpCodeActionHandler.cs

```diff
@@ -101,14 +95,14 @@ public override async Task<CommandOrCodeActionContainer> Handle(CodeActionParams
                         Diagnostics = new Container<Diagnostic>(),
                         Edit = new WorkspaceEdit(),
                         Command = Command.Create("omnisharp/executeCodeAction")
-                            .WithTitle(ca.Name)
                             .WithArguments(new CommandData()
                             {
                                 Uri = request.TextDocument.Uri,
                                 Identifier = ca.Identifier,
                                 Name = ca.Name,
                                 Range = request.Range,
                             })
+                            with { Title = ca.Name }
```

Comment:
We can't also with the arguments?

Reply:
as it stands yes and no..., the arguments is a [JArray](https://github.com/OmniSharp/csharp-language-server-protocol/blob/e0c9fc66da91ab389874f34ff9a5b1b18d932be6/src/Protocol/Models/Command.cs#L27-L28) so you'd have to do `with { Arguments = JArray.FromObject(...) }`